recipes/libdap4/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.18.0" %}
+
+package:
+  name: libdap4
+  version: {{ version }}
+
+source:
+  url: https://github.com/OPENDAP/libdap4/archive/version-{{ version }}.tar.gz
+  fn: version-{{ version }}.tar.gz
+  sha256: 373c60cf2c5c9eaf598558167aedbc3ef9a0d9b652dfbd96b4725637cf03f628
+
+build:
+  number: 0
+  skip: True  # [win]
+
+
+requirements:
+  build:
+    - automake
+    - autoconf
+    - libtool
+    - flex
+    - bison
+    - libgcc
```

Comment:
Do you need `libgcc` at build time? Usually we only add `defaults` `libgcc` at run time if we used `defaults` `gcc` at build time.